packages/replay/src/coreHandlers/handleClick.ts

```diff
@@ -125,7 +126,7 @@ export class ClickDetector implements ReplayClickDetector {
     }
 
     const newClick: Click = {
-      timestamp: breadcrumb.timestamp,
+      timestamp: timestampToS(breadcrumb.timestamp),
```

Comment:
should we use full name here?

```suggestion
      timestamp: timestampToSeconds(breadcrumb.timestamp),
```

Reply:
I used this to be parallel to `timestampToMs`, but 🤷 no strong feelings? WDYT?

Reply:
Yeah ToS just looks a bit funny to me, nbd